Makefile

```
@@ -4,13 +4,20 @@ PWD := $(shell pwd)
 LOCAL_BIN ?= $(PWD)/bin
 deployOnHub ?= false
 RELEASE_BRANCH ?= main
-OCM_API_COMMIT ?= $(shell awk '/open-cluster-management.io\/api/ {print $$2}' go.mod)
-DOCKER_URI ?= quay.io/stolostron
 ifneq ($(RELEASE_BRANCH), main)
   VERSION_TAG ?= latest-$(subst release-,,$(RELEASE_BRANCH))
 else
   VERSION_TAG ?= latest
 endif
+
+OCM_API_COMMIT ?= $(shell awk '/open-cluster-management.io\/api/ {print $$2}' go.mod)
+CALLER_REPO ?= stolostron
+ifeq ($(CALLER_REPO), open-cluster-management-io)
+  DOCKER_URI ?= quay.io/open-cluster-management
+else
+  DOCKER_URI ?= quay.io/stolostron
+endif
```

Nit: Might as well not have the else.
```suggestion
DOCKER_URI ?= quay.io/stolostron
ifeq ($(CALLER_REPO), open-cluster-management-io)
  DOCKER_URI ?= quay.io/open-cluster-management
endif
```